src/bip44.c

```diff
@@ -192,6 +192,20 @@ bool bip44_isOrdinarySpendingKeyPath(const bip44_path_t* pathSpec)
 	       bip44_containsAddress(pathSpec);
 }
 
+bool bip44_isMultisigSpendingKeyPath(const bip44_path_t* pathSpec)
+{
+	return bip44_hasMultisigWalletKeyPrefix(pathSpec) &&
+	       bip44_hasValidChainTypeForAddress(pathSpec) &&
+	       bip44_containsAddress(pathSpec);
+}
+
+bool bip44_isMintSpendingKeyPath(const bip44_path_t* pathSpec)
+{
+	return bip44_hasMintKeyPrefix(pathSpec) &&
+	       bip44_hasValidChainTypeForAddress(pathSpec) &&
+	       bip44_containsAddress(pathSpec);
+}
```

Comment:
This is totally wrong. See https://github.com/cardano-foundation/CIPs/blob/master/CIP-1855/CIP-1855.md for minting paths. I think we should rely on `bip44_classifyPath` in security policies (just deal with each possible path from the enum in a switch statement).

Reply:
I don't think in this particular case the classification would help, it would just bulk up the code. But true, I missed the memo on the 1855 paths. Fixed.